src/services/dataService.js

```diff
@@ -602,13 +602,17 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
 //            pollInterval: in milliseconds, how long to wait between polling the server
 //                    only applies if poll=true.  Default is 5000.
 //            http:   similar to .get, etc. at this point, only used to pass http.params for filtering
+//            suppressDigest: will skip the $apply & avoid triggering a digest loop
```

Comment:
I'm good for a rename.  I wonder if `manualApply` or `manualDigest` is preferable even, to make controller code easiest to read:

```JavaScript 
DataService.watch(resource, () = {
  // manually do updates
}, { skipApply: true });

DataService.watch(resource, () = {
  // manually do updates
}, { manuallyApply: true });
```
Thoughts?